### PR TITLE
Worked on the mobile navigation design

### PR DESCRIPTION
**Pull request by Sanjeev1308**

### CODE REVIEW

File: src/app/Layout/components/Header.tsx [hunk 1]
```diff
@@ -1,27 +1,54 @@
...
   height: 88,
   paddingLeft: 56,
   paddingRight: 56,
   display: "flex",
   justifyContent: "space-between",
   alignItems: "center",
-});
 
-const Logo = styled("img")({
+  [theme.breakpoints.down("sm")]: {
```
Comment (mohsenParvar) [inline src/app/Layout/components/Header.tsx]: use [mobile] here
we have the example in the governance page

File: src/app/Layout/index.tsx [hunk 2]
```diff
@@ -47,8 +54,12 @@ const SidebarWrapper = styled("div")({
...
 });
 
-const MainWrapper = styled("div")({
+const MainWrapper = styled("div")(({ theme }) => ({
   marginTop: 100,
   marginBottom: 50,
   marginLeft: "25%",
-});
+
+  [theme.breakpoints.down("sm")]: {
```
Comment (mohsenParvar) [inline src/app/Layout/index.tsx]: also here and everywhere you are using `theme.breakpoints.down("sm")`, use [mobile]

**Review by saito-sv — APPROVED** [review]
LGTM

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Review by mohsenParvar — APPROVED** [review]

**Review by bigwampa — CHANGES_REQUESTED** [review]
UI is failing to scale to mobile devices. Tried on both Android device and mobile device toolbar on desktop browser. Same display issue for navigatio
Screen Shot 2022-03-22 at 2 00 55 PM
n

**Comment by Sanjeev1308** [comment]
> UI is failing to scale to mobile devices. Tried on both Android device and mobile device toolbar on desktop browser. Same display issue for navigatio Screen Shot 2022-03-22 at 2 00 55 PM n

Please check with liquidity and rewards page

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-mobile-navigation.dev-app.axial.exchange

**Pull request merged**